DMTR-91.tex

```diff
@@ -1,564 +1,1334 @@
+The test campaign is considered in overall:
+PASS.\\[2\baselineskip]Missing elements on the Kubernetes Commons
+prevented us from building the Docker images in place and using a
+complete night of unique alerts. However, using pre-built Docker images
+with a smaller, repeated sample of alerts enabled the test to proceed as
+expected.
 
 
 \subsection{Recommended Improvements}
 \label{sect:recommendations}
 
-Re-execution with the missing components on the Kubernetes Commons.
+Re-execution with the missing components on the Kubernetes
+Commons.\\[2\baselineskip]Examination of the deserialized alerts in the
+logs is cumbersome and does not allow real verification of the alert
+content. Improved tools for counting alerts, comparing their contents to
+the sent alerts, and timing throughput would all improve the utility of
+this test.
 
-Examination of the deserialized alerts in the logs is cumbersome and does not allow real verification of the alert content. 
-Improved tools for counting alerts, comparing their contents to the sent alerts, and timing throughput would all improve the utility of this test.
 
+\newpage
+\section{Detailed Test Results}
+\label{sect:detailedtestresults}
 
 
+  \subsection{Test Cycle LVV-C3 }
 
-\newpage
+Open test cycle {\it \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testrun/LVV-C3}{LDM-503-5: Alert Distribution Validation}} in Jira.
 
-\section{Detailed Test Results}
-\label{sect:detailed}
+  LDM-503-5: Alert Distribution Validation\\
+  Status: Done
+
+  Test run with test cases in draft.
+
+
+  \subsubsection{Software Version/Baseline}
+    This test uses only the code in\\
+https://github.com/lsst-dm/alert\_stream/ , commit
+ef3c9e16c42a3265d4f6258ba6983be8b009e7c0
+
+
+  \subsubsection{Configuration}
+    \textbf{Documents}\\[2\baselineskip]This test report refers to the
+execution of tests described in the section 1.2 and documented in the
+\citeds{LDM-533} test specification issue
+2.0.\\[2\baselineskip]\textbf{Hardware}\\[2\baselineskip]This test
+activity shall be executed on the Kubernetes Commons at the LDF. As
+discussed in https://dmtn-028.lsst.io/ and https://dmtn-081.lsst.io/,
+the test machine should have at least 16 cores, 64 GB of memory and
+access to at least 1.5 TB of shared
+storage.\\[2\baselineskip]\textbf{Software}\\[2\baselineskip]This test
+uses only the code in\\
+https://github.com/lsst-dm/alert\_stream/ , commit
+ef3c9e16c42a3265d4f6258ba6983be8b009e7c0\\[2\baselineskip]\textbf{Input
+Data}\\[2\baselineskip]Input data for all tests was based on simulated
+alert packets\\[2\baselineskip]
 
 
-\subsection{LVV-T216(1.0): Installation of the Alert Distribution payloads}
-\label{sect:detail-lvv-t216}
+  \subsubsection{Test Cases in LVV-C3 Test Cycle}
+
+
+    \paragraph{Test Case LVV-T216 }\mbox{}\\
+
+Open  {\it \href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T216}{test case} } in Jira.
```

Comment:
Suggest this should be
```
\textit{\href{https://jira.lsstcorp.org/secure/Tests.jspa#/testCase/LVV-T216}{test case LVV-T216}}
```
(the change from `\it` to `\textit` is just good style; the important thing is to capture the test name in the link).

Reply:
I will report this as improvement in docsteady.